fuse/server_darwin.go

```diff
@@ -30,3 +32,35 @@ func (ms *Server) systemWrite(req *request, header []byte) Status {
 	}
 	return ToStatus(err)
 }
+
+// for a stream connection we need to have a single reader
+var readLock sync.Mutex
```

Comment:
Maybe that ship has sailed, but why not SOCK_DGRAM ?

Reply:
SOCK_DGRAM sounds like a change for the next version of fuse-t.